tools/accuracy_checker/accuracy_checker/evaluators/custom_evaluators/cocosnet_evaluator.py

```diff
@@ -332,6 +335,15 @@ def configure_intermediate_metrics_results(config):
     def dataset_size(self):
         return self.dataset.size
 
+    def _write_results_to_csv(self, csv_file, ignore_results_formatting, metric_interval, processing_info):
```

Comment:
I believe this method behavior is the same for all evaluators, please move to base class